wooting-macro-backend/src/lib.rs

```diff
@@ -281,13 +300,22 @@ async fn execute_macro(macros: Macro, channel: UnboundedSender<rdev::EventType>)
 /// Puts a mandatory 0-20 ms delay between each macro execution (depending on the platform).
 fn keypress_executor_sender(mut rchan_execute: UnboundedReceiver<rdev::EventType>) {
     loop {
-        plugin::util::send(&rchan_execute.blocking_recv().unwrap());
+        let received_event = match &rchan_execute.blocking_recv() {
+            Some(event) => *event,
+            None => {
+                error!("Failed to receive an event!");
+                thread::sleep(time::Duration::from_millis(200));
+                continue;
+            }
+        };
+        plugin::util::direct_send_event(&received_event)
+            .unwrap_or_else(|err| error!("Error directly sending an event to keyboard: {}", err));
 
-        //Windows requires a delay between each macro execution.
+        //MacOS requires some strange delays so putting it here just in case.
         #[cfg(any(target_os = "macos", target_os = "linux"))]
         thread::sleep(time::Duration::from_millis(10));
```

Comment:
"MacOS requires..."
`target_os = "linux"`

Reply:
Fixed also in: [5d256ec](https://github.com/WootingKb/wooting-macros/pull/194/commits/5d256ecf2bf5fb0a8a4c1027963d6ec4b0fe7aa8)